play-json/shared/src/test/scala/play/api/libs/json/FormatEnumReadsSpec.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package play.api.libs.json
+
+import org.scalatest._
+
+
+class FormatEnumReadsSpec extends WordSpec with MustMatchers {
```

Comment:
Seems I was clear enough. I mean that it should either be the initial code for a `FormatSharedSpec` for all the tests about `Format`, or, what I would prefer, be put in the existing `ReadSharedSpec` and `WriteSharedSpec`.

I don't see there need/benefit to separate those tests.

Reply:
@cchantep from my point of view, there is no need to put them together.
If you would like to, you can update this PR

Reply:
For the few amount of code, there is no need to introduce more files, and make the tests more complex

Reply:
@cchantep done, I hope that TestEnums could stay there and we won't duplicate them (: